backend/controllers/prisonerProfile/prisonerProfileRedirect.ts

```diff
@@ -11,9 +10,7 @@ export default ({ path, handler }) => {
       `Old prisoner profile disabled from: ${config.app.prisonerProfileRedirect.oldPrisonerProfileInaccessibleFrom}`
     )
 
-    const redirectEnabled = isRedirectCaseLoad(activeCaseLoadId)
-
-    if (activeCaseLoadId && redirectEnabled) {
+    if (activeCaseLoadId) {
```

Comment:
Without the concept of exemptions, we can see clearer that the Prisoner Profile redirect is now solely on whether the user has an activeCaseLoad set!